### PR TITLE
fix: Redirect to collection on self-hosted

### PR DESCRIPTION
**Pull request by iamsaumya**
I realised that I was not redirected to the collection when launching the app for the first time 😄

### CODE REVIEW

File: server/utils/authentication.js [hunk 1]
```diff
@@ -99,6 +99,23 @@ export async function signIn(
       httpOnly: false,
       expires,
     });
-    ctx.redirect(`${team.url}/home${isNewUser ? "?welcome" : ""}`);
+
+    const [team, collection, view] = await Promise.all([
+      Team.findByPk(user.teamId),
+      Collection.findOne({
+        where: { teamId: user.teamId },
+        order: [["index", "ASC"]],
```
Comment (iamsaumya) [inline server/utils/authentication.js:107]: Assures that we fetch the top-most collection.

**Comment by tommoor** [comment]
Yea, I knew this was a difference between the two but honestly I just didn't want to replicate the logic in two places. First-run self-hosted experience isn't something I care too much about 🙈

File: server/utils/authentication.js (hunk 1 above)
Comment (iamsaumya) [reply server/utils/authentication.js:107]: Ohh! Copy paste mistake it is.

**Comment by iamsaumya** [comment]
I see. So, would you like to go ahead with PR, or should I close it?

**Comment by tommoor** [comment]
It's fine, you fixed my laziness

**Review by tommoor — APPROVED** [review]

**Pull request merged**